app/views/allocations/history.html.erb

```diff
@@ -8,7 +8,8 @@
       <h2 class="govuk-heading-m"><%= prison_title(prison) %></h2>
       <div class="moj-timeline">
           <% allocations.each_with_index do |allocation, i| %>
-            <% if allocation.event == 'allocate_primary_pom' %>
+<!-- It is size-1 because the list is displayed backwards -->
+            <% if allocation.event == 'allocate_primary_pom' || i == allocations.size-1 %>
```

Comment:
`i == allocations.size-1` might be more readable if written as:

```ruby
allocation == allocations.last
```

i.e. "is the current allocation the last one in the array?"

This works because of the [Ruby method `.last`](https://ruby-doc.org/core-2.6.3/Array.html#method-i-last), which returns the last element of an Array.

Reply:
@ollietreend thanks Ollie. Yes that is much more readable.